### PR TITLE
NEPT-539: Upgrade entity_translation

### PR DESCRIPTION
**Pull request by imanoleguskiza**

### CODE REVIEW

**Comment by drupol** [comment]
Nice little perfomance improvement there 👍

File: resources/multisite_drupal_standard.make [hunk 1]
```diff
@@ -187,11 +187,20 @@ projects[email][version] = "1.3"
...
+projects[entity_translation][download][type] = git
 projects[entity_translation][subdir] = "contrib"
-projects[entity_translation][version] = "1.0-beta5"
 ; Issue #1707156 : Workbench Moderation integration
 ; https://www.drupal.org/node/1707156
-projects[entity_translation][patch][] = https://www.drupal.org/files/issues/workbench_moderation-1707156-47.patch
+projects[entity_translation][patch][] = https://www.drupal.org/files/issues/workbench_moderation-1707156-63.patch
+; Update to entity_translation beta6
+; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-539
+projects[entity_translation][patch][] = https://www.drupal.org/files/issues/entity_translation-strict_warning_only_variables_should_be_passed_by_reference-2859223-2.patch
```
Comment (gillesdeudon) [inline resources/multisite_drupal_standard.make]: @imanoleguskiza, could you put for each patch reference a comment the URL to the Drupal issue from where the patch comes like for workbench_moderation-1707156-63.patch?

**Comment by drupol** [comment]
👍

**Comment by dxvargas** [comment]
👍

**Comment by drupol** [comment]
❤️

**Pull request merged**